src/main/java/se/diabol/jenkins/pipeline/DeliveryPipelineView.java

```diff
@@ -486,29 +488,28 @@ protected static String withoutFolderPrefix(final String projectName) {
             }
             LOG.fine("Returning: " + components);
             error = null;
-            for(int i = 0; i< components.size(); i++) {
-                components.get(i).setComponentNumber(i+1);
```

Comment:
According to previous design, I am adding a number to each Component by looping after all components created.
But this adding number to component moved to at the time of creation of Component (i.e input parameter in constructor). So this has been removed here.

The moved piece is in above comment section.